releasenotes/notes/bosonic-operator-and-mapper-45bfde873f092681.yaml

```
@@ -0,0 +1,24 @@
+---
+features:
+  - |
+    Introduced a new feature that implements the bosonic operators ``BosonicOp``.
+    Its functionalities are analogous to the ``FermionicOp`` It should be used to represent a bosonic
+    operator, so if one wants to represent the boson number operator it should do for example::
+
+      from qiskit_nature.second_q.operators import BosonicOp
+      bosonic_op = BosonicOp({'+_0 -_0': 1}, num_modes=1)
```

For the code block to look like python code, you should follow this syntax: (the empty line between `..code-block::python` and the snippet is key)

```suggestion
    operator, so if one wants to represent the boson number operator it should do for example:
    
    .. code-block:: python
    
      from qiskit_nature.second_q.operators import BosonicOp
      bosonic_op = BosonicOp({'+_0 -_0': 1}, num_modes=1)
```

---

Solved